test/custom_resource_crud/custom_resource_crud_c3_test.go

```diff
@@ -0,0 +1,296 @@
+			testenv.VerifyCPULimits(deployment, testenvInstance.GetName(), SearchHeadPodName, defaultCPULimits)
+			SearchHeadPodName = fmt.Sprintf(testenv.SearchHeadPod, deployment.GetName(), 1)
+			testenv.VerifyCPULimits(deployment, testenvInstance.GetName(), SearchHeadPodName, defaultCPULimits)
+			SearchHeadPodName = fmt.Sprintf(testenv.SearchHeadPod, deployment.GetName(), 2)
+			testenv.VerifyCPULimits(deployment, testenvInstance.GetName(), SearchHeadPodName, defaultCPULimits)
+
+			// Change CPU limits to trigger CR update
+			shc := &enterprisev1.SearchHeadCluster{}
+			instanceName = fmt.Sprintf("%s-shc", deployment.GetName())
+			err = deployment.GetInstance(instanceName, shc)
+			if err != nil {
+				testenvInstance.Log.Error(err, "Unable to fetch Search Head Cluster deployment")
+			}
+
+			shc.Spec.Resources.Limits = corev1.ResourceList{
+				"cpu": resource.MustParse(newCPULimits),
+			}
+			err = deployment.UpdateCR(shc)
+			Expect(err).To(Succeed(), "Unable to deploy Search Head Cluster with updated CR")
+
+			// Verify Search Head Cluster is updating
+			testenv.VerifySearchHeadClusterPhase(deployment, testenvInstance, splcommon.PhaseUpdating)
+
+			// Verify Search Head go to ready state
+			testenv.SearchHeadClusterReady(deployment, testenvInstance)
+
+			// Verify CPU limits after updating the CR
+			SearchHeadPodName = fmt.Sprintf(testenv.SearchHeadPod, deployment.GetName(), 0)
+			testenv.VerifyCPULimits(deployment, testenvInstance.GetName(), SearchHeadPodName, newCPULimits)
+			SearchHeadPodName = fmt.Sprintf(testenv.SearchHeadPod, deployment.GetName(), 1)
+			testenv.VerifyCPULimits(deployment, testenvInstance.GetName(), SearchHeadPodName, newCPULimits)
+			SearchHeadPodName = fmt.Sprintf(testenv.SearchHeadPod, deployment.GetName(), 2)
+			testenv.VerifyCPULimits(deployment, testenvInstance.GetName(), SearchHeadPodName, newCPULimits)
+		})
+	})
+
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
+		It("crcrud: can verify IDXC, CM and SHC PVCs are correctly deleted after the CRs deletion", func() {
+
+			// Deploy Single site Cluster and Search Head Clusters
+			err := deployment.DeploySingleSiteCluster(deployment.GetName(), 3, true /*shc*/)
+			Expect(err).To(Succeed(), "Unable to deploy cluster")
+
+			// Ensure that the Cluster Master goes to Ready phase
+			testenv.ClusterMasterReady(deployment, testenvInstance)
+
+			// Ensure Indexers go to Ready phase
+			testenv.SingleSiteIndexersReady(deployment, testenvInstance)
+
+			// Ensure Search Head Cluster go to Ready phase
+			testenv.SearchHeadClusterReady(deployment, testenvInstance)
+
+			// Verify MC Pod is Ready
+			testenv.MCPodReady(testenvInstance.GetName(), deployment)
+
+			// Verify RF SF is met
+			testenv.VerifyRFSFMet(deployment, testenvInstance)
+
+			// Define Pvcs
+			DeployerEtcPvc := "pvc-etc-splunk-" + deployment.GetName() + "-shc-deployer-0"
```

Comment:
`DeployerEtcPvc` vs `deployerEtcPvc`. Use `DeployerEtcPvc` only if these variables are exported out of this file.